code/modules/mining/lavaland/necropolis_chests.dm

```diff
@@ -69,8 +75,7 @@
 		if(27)
 			new /obj/item/borg/upgrade/modkit/lifesteal(src)
 			new /obj/item/bedsheet/cult(src)
-		if(28)
```

Comment:
update the rand!

Reply:
can we just talk about how cancer this thing looks